bash_103_combining_commands/bash_103_combining_commands.md

```diff
@@ -2,7 +2,7 @@
 
 author:   Elizabeth Drellich and Nicole Feldman
 email:    drelliche@chop.edu and feldmanna@chop.edu
-version: 1.1.0
+version: 1.3.0
```

Comment:
```suggestion
version: 1.2.0
```

This should just be incremented by 1, right?

Reply:
I think it has already been incremented in another PR, which is why there is a conflict... current version on main is 1.2.0

Reply:
Ah makes sense!

Reply:
Ah makes sense!